community/consistency-check-legacy/src/main/java/org/neo4j/legacy/consistency/checking/full/FullCheck.java

```diff
@@ -161,7 +163,7 @@ static DiffRecordAccess recordAccess( StoreAccess store )
         T[] records = (T[]) Array.newInstance( type, (int) store.getHighId() );
         for ( int i = 0; i < records.length; i++ )
         {
-            records[i] = store.forceGetRecord( i );
+            records[i] = store.getRecord( i, store.newRecord(), FORCE );
```

Comment:
Don't really care about characters. We should care about object instantiation... having it explicit does just that.